ci-operator/step-registry/cucushift/upgrade/mirror-images/by-oc-mirror/cucushift-upgrade-mirror-images-by-oc-mirror-ref.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/upgrade/mirror-images/by-oc-mirror/cucushift-upgrade-mirror-images-by-oc-mirror-ref.yaml",
```

Comment:
Let us complete the fresh install step firstly, https://github.com/openshift/release/tree/master/ci-operator/step-registry/mirror-images/by-oc-adm, we can follow up the upgrade one later.